build_tools/README.md

```diff
@@ -27,3 +29,9 @@ $ cmake .. -GNinja \
    -DMLIR_DIR=${PWD}/../llvm-build/lib/cmake/mlir
 $ ninja check-stablehlo
 ```
+
+## Bazel build
+
+To build the code in this repository,
+
+`$ bazel build :all`
```

Comment:
Also add instructions for running tests?

Reply:
At the moment we are not enabling the lit tests using Bazel. I am waiting for iree's lit bazel macros to get up-streamed to llvm.

Reply:
We can track/addressed this via separate  issue.

Reply:
What do we do with `build_tools/bazel/glob_lit_test.bzl` in the meanwhile?